.drone.yml

```diff
@@ -11,7 +11,8 @@ workspace:
 
 steps:
   - name: test
-    image: golangci/golangci-lint:v1.38.0-alpine
```

Comment:
The drone test was failing due to the slog package wasn't found by golangci. Probably has to do with that slog was introduced into golang in 1.21. [v1.38.0](https://github.com/golangci/golangci-lint/releases/tag/v1.38.0) uses go v1.14.

Reply:
The go.mod should also be updated in this repository then.